src/test/java/com/google/edith/UserStatsServletTest.java

```diff
@@ -0,0 +1,81 @@
+package com.google.edith;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.edith.servlets.UserInsightsInterface;
+import com.google.edith.servlets.UserStatsServlet;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import java.io.BufferedReader;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public final class UserStatsServletTest {
+
+  private DatastoreService datastore;
```

Comment:
could this not be final and initialized here as well?

Reply:
I can't initialize it because it has to be initialized after the test helper is set up. And I don't know why but I keep getting an error when I try to make it final.